packages/examples/component-library/package.json

```diff
@@ -29,6 +29,7 @@
     "@pixeloven/cli": "^7.0.0-alpha.0",
     "@pixeloven/cli-addon-storybook": "^7.0.0-alpha.1",
     "@pixeloven/cli-core": "^7.0.0-alpha.0",
+    "@storybook/addon-docs": "6.2.7",
```

Comment:
Whoops, pretty sure I don't need this, but it was complaining it didn't exist. Lemme remove it and see

Reply:
PR looks good just want to remove this if we can.

Reply:
Ohhh, I remember why it's necessary now, you have to import the doc components into the MDX stories

```
import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs/blocks';
```